dataactbroker/handlers/aws/session.py

```diff
@@ -254,7 +254,7 @@ def save_session(self, app, session, response):
             else:
                 expiration = datetime.utcnow() + timedelta(seconds=SessionTable.TIME_OUT_LIMIT)
         if(not "_uid" in session):
-            session["_uid"] = _create_identifier()
+            session["_uid"] = "{}|{}".format(_create_identifier(),uuid4())
```

Comment:
Small detail, but rather than duplicating the logic, could we call `self.resetId(session)`?